README.md

```diff
@@ -165,7 +165,7 @@ Create the PR, using the upstream (parent) branch from `.git/machete` as the bas
 git machete github create-pr [--draft]
 ```
 
-Synchronize with the remote repository: delete untracked and unmanaged branches and checkout your open PRs: <br/>
+Synchronize with the remote repository: delete untracked and unmanaged branches with no downstream branch and also checkout your open PRs: <br/>
```

Comment:
```suggestion
Synchronize with the remote repository: delete unmanaged branches with no downstream branch, untracked managed branches and and also checkout your open PRs: <br/>
```

Reply:
(IIUC that's precisely what happens)

Reply:
(once the final form is agreed upon, reflect in all other places)

Reply:
@PawelLipski Synchronize with the remote repository: delete unmanaged branches with no downstream branch, delete untracked managed branches and also checkout your open PRs. Is this ok?

Reply:
Whoops right, this way ☝🏻

Reply:
My original suggestion is wrong ofc